apps/next/util/s3/download.ts

```diff
@@ -0,0 +1,17 @@
+import S3 from "aws-sdk/clients/s3";
```

Comment:
Ya we can, I kinda prefer more small files over one large one but for this I think we are good to combine. I didn't extract delete since we only used it in one obscure place (deleting the user).